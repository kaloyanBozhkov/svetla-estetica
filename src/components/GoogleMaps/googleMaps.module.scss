.googleMapsWrapper {
    width: 100%;
    height: 45.2rem;
}