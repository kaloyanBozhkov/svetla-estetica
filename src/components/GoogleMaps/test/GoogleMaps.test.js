import { shallow } from 'enzyme'
import React from 'react'
import GoogleMaps from '../GoogleMaps'

describe('testing GoogleMaps', () => {
    it('should render component', () => {
        expect(shallow(<GoogleMaps />)).toMatchSnapshot()
    })
})
