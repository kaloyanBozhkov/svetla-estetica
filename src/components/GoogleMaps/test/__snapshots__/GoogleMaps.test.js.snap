// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing GoogleMaps should render component 1`] = `
<iframe
  className="googleMapsWrapper"
  frameBorder="0"
  src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d493.0138478039089!2d9.602949872562638!3d45.6526797088034!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4781520a049a6ae5%3A0xc3a83e5635025f72!2sSvetla+Estetica!5e0!3m2!1sen!2sus!4v1494076022105"
  title="Google Maps - Svetla Estetica Dalmine"
/>
`;
